EIPS/eip-3651.md

```diff
@@ -0,0 +1,38 @@
+---
+eip: 3651
+title: Warm COINBASE
+author: wjmelements.eth <wjmelements@gmail.com> @wjmelements
```

Comment:
```suggestion
author: William Morriss (@wjmelements), William Morriss <wjmelements@gmail.com>
```
If you don't want your full name we can replace it with `wjmelements.eth` but keep in mind that git history will have it so you won't really benefit from any privacy this wayw.